vktrace/vktrace_common/vktrace_filelike.h

```diff
@@ -37,6 +37,7 @@ typedef struct FileLike FileLike;
 typedef struct FileLike {
     enum { File, Socket } mMode;
     FILE* mFile;
+    size_t mMemLen;
```

Comment:
mMemLen doesn't describe what this variable holds. Maybe change it to mFileLen?

Reply:
Yes, mFileLen is more clear to describe this variable.  Will update it.